mhkit/tests/test_wave.py

```diff
@@ -41,6 +42,25 @@ def test_bretschneider_spectrum(self):
         
         self.assertLess(errorHm0, 0.01)
         self.assertLess(errorTp0, 0.01)
+
+    def test_surface_elevation(self):
+        S = wave.resource.jonswap_spectrum(self.f, self.Tp, self.Hs)
+        eta = wave.resource.surface_elevation(S, self.t)
+        dt = self.t[1] - self.t[0]
+        df = self.f[1] - self.f[0]
+        Sn = wave.resource.elevation_spectrum(eta, 1/dt, int(np.floor(self.t[-1]/df)))
+
+        m0 = wave.resource.frequency_moment(S,0).m0.values[0]
+        m0n = wave.resource.frequency_moment(S,0).m0.values[0]
+        errorm0 = np.abs((m0 - m0n)/m0)
+
+        self.assertLess(errorm0, 0.01)
+
+        m1 = wave.resource.frequency_moment(S,1).m1.values[0]
+        m1n = wave.resource.frequency_moment(S,1).m1.values[0]
```

Comment:
It looks like m0n and m1n should be calculated with Sn.
Can we also compare the S and Sn directly?

Reply:
Quite right! Updating this the moments, for the specific wave case being used for the test, I have to increase the error tolerance to 5%.

You could certainly also compare the spectra directly, I just wasn't sure of an elegant way to do this...

This is my first pull request... let me know how we should proceed.